### ISSUE TITLE
NoSuchMethodError: The method 'cast' was called on null.

### ISSUE THREAD

**Issue by blue7wings** [FlutterGen/flutter_gen#21]
hi, I can't generate asset, maybe cause by code error.
![image](https://user-images.githubusercontent.com/4016570/92426179-d4d30900-f1bb-11ea-8636-6200d9c5aa27.png)
Dart SDK version: 2.9.0-21.10.beta (beta) (Tue Jul 21 10:46:30 2020 +0200) on "windows_x64"

**Comment by lifenautjoe** [comment]
I'm still seeing this issue.

```
[SEVERE]
NoSuchMethodError: The method 'cast' was called on null.
Receiver: null
Tried calling: cast<String>()
```

No files generated.

**Comment by lifenautjoe** [comment]
Can we please keep this open @wasabeef ? I know you've already fixed it but it might be useful for people to know that the package won't work until that version is released.

**Comment by wasabeef** [comment]
@lifenautjoe 
Sorry I'm not yet releasing the fix version of 1.0.1.
Can I ask you? Which uses your PC? macos?ubuntu?windows?

**Comment by lifenautjoe** [comment]
Thanks @wasabeef :-)

I'm on MacOS.

Here's my Flutter doctor in case it helps.


````
[✓] Flutter (Channel unknown, 1.20.3, on macOS 11.0 20A5354i, locale en-NL)
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 12.0)
[✓] Android Studio (version 4.0)
[✓] IntelliJ IDEA Ultimate Edition (version 2020.1.3)
[✓] VS Code (version 1.48.0)
[!] Connected device
````

I've also tried using the `main` branch which has the patch merged but still see the same error. I'm guessing you already know though if you're asking me for my device details :-)

**Comment by lifenautjoe** [comment]
Correction, the `main` branch does work. Sorry, caching issue.

**Comment by wasabeef** [comment]
@lifenautjoe 
Thanks.

I will release v1.0.1 within an hour.

**Comment by wasabeef** [comment]
@blue7wings @nekocode @lifenautjoe 

I appreciate all the support.
Just released v1.0.1.

```
# Homebrew
brew upgrade fluttergen
```

```
# Dart command-line
pub global activate flutter_gen
```

```
# build_runner
flutter pub get
```

**Comment by lifenautjoe** [comment]
Great job @wasabeef ! Thanks for writing this package and the quick fix! 🥳
